kale/pipeline/multi_domain_adapter.py

```diff
@@ -333,3 +337,118 @@ def cls_discrepancy(self, x):
                 cls_disc += torch.mean(torch.abs(cls_disc_))
 
         return cls_disc * 2 / (n_domains * (n_domains - 1))
+
+
+class CoIRLS(BaseEstimator, ClassifierMixin):
+    """Covariate-Independence Regularized Least Squares (CoIRLS)
+
+    Args:
+        kernel (str, optional): {"linear", "rbf", "poly"}. Kernel to use. Defaults to "linear".
+        kernel_kwargs (dict or None, optional): Hyperparameter for the kernel. Defaults to None.
+        alpha (float, optional): Hyperparameter of the l2 (Ridge) penalty. Defaults to 1.0.
+        lambda_ (float, optional): Hyperparameter of the covariate dependence.  Defaults to 1.0.
+
+    Reference:
+        [1] Zhou, S., 2022. Interpretable Domain-Aware Learning for Neuroimage Classification (Doctoral dissertation,
+            University of Sheffield).
+        [2] Zhou, S., Li, W., Cox, C.R., & Lu, H. (2020). Side Information Dependence as a Regularizer for Analyzing
+            Human Brain Conditions across Cognitive Experiments. AAAI 2020, New York, USA.
+    """
+
+    def __init__(self, kernel="linear", kernel_kwargs=None, alpha=1.0, lambda_=1.0):
+        super().__init__()
+        self.kernel = kernel
+        self.model = None
+        self.alpha = alpha
+        self.lambda_ = lambda_
+        if kernel_kwargs is None:
+            self.kernel_kwargs = dict()
+        else:
+            self.kernel_kwargs = kernel_kwargs
+        self.device = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")
+        self.losses = {"ovr": [], "pred": [], "code": [], "reg": []}
+        self.x = None
+        self._lb = LabelBinarizer(pos_label=1, neg_label=-1)
```

Comment:
lb: can you make it more readable (lb stands for pound https://en.wikipedia.org/wiki/Pound_(mass)?